cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -98,6 +98,7 @@ ocv_optimization_process_obsolete_option(ENABLE_POPCNT POPCNT ON)
 ocv_optimization_process_obsolete_option(ENABLE_AVX AVX ON)
 ocv_optimization_process_obsolete_option(ENABLE_AVX2 AVX2 ON)
 ocv_optimization_process_obsolete_option(ENABLE_FMA3 FMA3 ON)
+ocv_optimization_process_obsolete_option(ENABLE_LASX LASX ON)
```

Comment:
> obsolete_option

Obsolete approach should not be used anymore.
Won't fix.

Refer to this page how to enable optimizations during the build: https://github.com/opencv/opencv/wiki/CPU-optimizations-build-options

Reply:
Thank you very much for your reminder. My understanding of opencv community development is outdated. Then I will close this PR.